app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee

```diff
@@ -38,9 +38,9 @@ class App.WarehouseReports.Rrh.Makeup
           ticks: 4
       legend:
         show: true
-        position: 'right'
+        position: 'bottom'
```

Comment:
This didn't solve the problem for me -- do we need to to estimate the size somehow, or is this just bad data for me (I see a NaN)?

<img width="1050" alt="image" src="https://user-images.githubusercontent.com/33072305/113621479-a9b57880-9629-11eb-970e-968a3e4c5591.png">

Reply:
I _think_ that's a data problem, I don't have quite so many projects, but with a sizable number it seems to render fine.  Happy to up the size too.

Reply:
![Screen Shot 2021-04-05 at 4 47 36 PM](https://user-images.githubusercontent.com/1346876/113625291-bd171280-962e-11eb-8ee3-d4dc5a0635c1.png)
![Screen Shot 2021-04-05 at 4 48 01 PM](https://user-images.githubusercontent.com/1346876/113625292-bdafa900-962e-11eb-9861-9b559d3287ba.png)